exercises/all-your-base/all_your_base_test.go

```diff
@@ -1,177 +1,21 @@
-		outputDigits: []uint64{1, 0, 1},
-	},
-	{
-		description:  "binary to multiple decimal",
-		inputBase:    2,
-		inputDigits:  []uint64{1, 0, 1, 0, 1, 0},
-		outputBase:   10,
-		outputDigits: []uint64{4, 2},
-	},
-	{
-		description:  "decimal to binary",
-		inputBase:    10,
-		inputDigits:  []uint64{4, 2},
-		outputBase:   2,
-		outputDigits: []uint64{1, 0, 1, 0, 1, 0},
-	},
-	{
-		description:  "trinary to hexadecimal",
-		inputBase:    3,
-		inputDigits:  []uint64{1, 1, 2, 0},
-		outputBase:   16,
-		outputDigits: []uint64{2, 10},
-	},
-	{
-		description:  "hexadecimal to trinary",
-		inputBase:    16,
-		inputDigits:  []uint64{2, 10},
-		outputBase:   3,
-		outputDigits: []uint64{1, 1, 2, 0},
-	},
-	{
-		description:  "15-bit integer",
-		inputBase:    97,
-		inputDigits:  []uint64{3, 46, 60},
-		outputBase:   73,
-		outputDigits: []uint64{6, 10, 45},
-	},
-	{
-		description:  "empty list",
-		inputBase:    2,
-		inputDigits:  []uint64{},
-		outputBase:   10,
-		outputDigits: []uint64{0},
-		error:        nil,
-	},
-	{
-		description:  "single zero",
-		inputBase:    10,
-		inputDigits:  []uint64{0},
-		outputBase:   2,
-		outputDigits: []uint64{0},
-	},
-	{
-		description:  "multiple zeros",
-		inputBase:    10,
-		inputDigits:  []uint64{0, 0, 0},
-		outputBase:   2,
-		outputDigits: []uint64{0},
-	},
-	{
-		description:  "leading zeros",
-		inputBase:    7,
-		inputDigits:  []uint64{0, 6, 0},
-		outputBase:   10,
-		outputDigits: []uint64{4, 2},
-	},
-	{
-		description:  "invalid positive digit",
-		inputBase:    2,
-		inputDigits:  []uint64{1, 2, 1, 0, 1, 0},
-		outputBase:   10,
-		outputDigits: nil,
-		error:        ErrInvalidDigit,
-	},
-	{
-		description:  "first base is one",
-		inputBase:    1,
-		inputDigits:  []uint64{},
-		outputBase:   10,
-		outputDigits: nil,
-		error:        ErrInvalidBase,
-	},
-	{
-		description:  "second base is one",
-		inputBase:    2,
-		inputDigits:  []uint64{1, 0, 1, 0, 1, 0},
-		outputBase:   1,
-		outputDigits: nil,
-		error:        ErrInvalidBase,
-	},
-	{
-		description:  "first base is zero",
-		inputBase:    0,
-		inputDigits:  []uint64{},
-		outputBase:   10,
-		outputDigits: nil,
-		error:        ErrInvalidBase,
-	},
-	{
-		description:  "second base is zero",
-		inputBase:    10,
-		inputDigits:  []uint64{7},
-		outputBase:   0,
-		outputDigits: nil,
-		error:        ErrInvalidBase,
-	},
-}
-
-func digitsEqual(a, b []uint64) bool {
-	if len(a) != len(b) {
-		return false
-	}
-
-	for i := 0; i < len(a); i++ {
-		if a[i] != b[i] {
-			return false
-		}
-	}
-
-	return true
-}
+import (
+	"reflect"
+	"testing"
+)
 
 func TestConvertToBase(t *testing.T) {
 	for _, c := range testCases {
-		output, err := ConvertToBase(c.inputBase, c.inputDigits, c.outputBase)
-		if err != c.error {
-			t.Fatalf(`FAIL: %s
-	Expected error: %v
-	Got: %v`, c.description, c.error, err)
-		}
-
-		if !digitsEqual(c.outputDigits, output) {
+		output := ConvertToBase(c.inputBase, c.inputDigits, c.outputBase)
```

Comment:
Dropping the explicit error return value and collapsing all the errors to a nil slice seems a bit atypical.